study-builder/studies/rarex/snippets/consent-section2-risks-self.conf

```diff
@@ -71,12 +75,80 @@
             "language": "en",
             "text": """
               It is possible that someone who has access to your identifiable data could use or share it in a
-              way that could make it harder for you or your family to get or keep a job or insurance. There are
-              laws against the misuse of genetic data, but they may not protect your data in all cases.
+              way that could make it harder for you or your family to get or keep a job or insurance. There
+              are laws against the misuse of genetic data, but they may not protect your data in all cases.
             """
           }
         ]
-      }
+      },
+      {
+        "name": "p5",
+        "translations": [
+          {
+            "language": "en",
+            "text": """
+              As part of this research, you may be required to use one or more of the following to collect research
+              data: a phone or web app/ site, online/electronic questionnaires/surveys, or a device that tracks
+              information about you.
+            """
+          }
+        ]
+      },
+      {
+        "name": "p6",
+        "translations": [
+          {
+            "language": "en",
+            "text": """
+              While using these electronic tools, information about you may be collected and shared with the
+              researchers or people outside of the study. This data might include personal health information,
+              location, call logs, text message history, web browsing history, or social media use. A complete
+              description of the data collection and data sharing for an app, online/electronic
+              questionnaires/surveys, or device can commonly be found in the Terms of Use, End User License
+              Agreement, or Privacy Policy. If you would like to read these documents, request a copy or instructions
+              about how to access this information from the study staff. While the Terms of Use, End User License
+            """
+          }
+        ]
+      },
+      {
+        "name": "p7",
+        "translations": [
+          {
+            "language": "en",
+            "text": """
+              Agreement, or Privacy Policy may include statements limiting your rights if you are harmed as a result of
```

Comment:
This should be one paragraph? E.g. `... End User License Agreement, or Privacy Policy ...`.

Reply:
as far as I see they are separate texts as it is shown in the pdf files which are attached to this task

Reply:
It looks separate in the PDF but that's only because they're on different pages. They're suppose to be a single paragraph.